.devcontainer/devcontainer.json

```diff
@@ -1,28 +1,16 @@
 {
-	"name": "sense-base ROS2 with cuda-12.5.0-devel-ubuntu22.04",
+	"name": "base ROS2 with cuda-12.6.0-devel-ubuntu24.04",
 	"build": {
 		"dockerfile": "Dockerfile",
 		"context": ".."
 	},
-	"workspaceMount": "source=${localWorkspaceFolder},target=/workspace/sense-base,type=bind,consistency=cached",
-	"workspaceFolder": "/workspace/sense-base",
-	"mounts": [
-		{
-			"source": "${localWorkspaceFolder}/../sense_eeg",
-			"target": "/workspace/sense-base/sense_eeg",
-			"type": "bind"
-		},
-		{
-			"source": "${localWorkspaceFolder}/../sense_msgs",
-			"target": "/workspace/sense-base/sense_msgs",
-			"type": "bind"
-		}
-	],
+	"workspaceMount": "source=${localWorkspaceFolder}/workspace,target=/workspace/,type=bind", 
+	"workspaceFolder": "/workspace",
 	"containerEnv": {
 		"DISPLAY": "${localEnv:DISPLAY}",
-		"XAUTHORITY": "/root/.Xauthority"
-	},
-	"privileged": true,
```

Comment:
we need the `privileged` flag for the time being for the camera work

Reply:
sorted out here [8f91458](https://github.com/sense-base/base/pull/38/commits/8f914581170cff80ae67d45fc9b3eddbf18be6e2)